src/components/atoms/Input.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface Props {
+    type: string,
+    className: string,
+    setState: React.Dispatch<React.SetStateAction<string>>
```

Comment:
mets un onChange plutôt que de passer ton state dans la vu récupère uniquement la valeur

```suggestion
    onChange: (value: string) => void;
```